.github/workflows/main.yml

```diff
@@ -2,37 +2,37 @@ name: CI
 
 on:
   push:
-    branches: [ master ]
+    branches: [master]
   pull_request:
-    branches: [ master ]
+    branches: [master]
 
 jobs:
   build-and-test:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      
+
       - name: Setup Go
-        uses: actions/setup-go@v2
+        uses: actions/setup-go@v4
         with:
-          go-version: '1.15.3' 
-      
+          go-version-file: go.mod
```

Comment:
Apparently necessary to prevent [this](https://github.com/starboard-nz/orb/actions/runs/5106929731)

Reply:
Very strange...that deprecation should not happen when explicitly specifying the Go version like this. Oh well, time to update anyway.